### ISSUE TITLE
ESTOP not working properly

### ISSUE THREAD

**Issue by Klimbim1** [FarmBot/farmbot_os#325]
Hi all,

the estop button isn´t working properly. I needed to press several times in order to execute an emergency stop. That behavior changed from 3.1.6. where it was working properly.
Even more if I want to unlock again the bot says:

```
UNKNOWN ERROR HAPPEND EXECUTING CELERYSCRIPT.
4:35 PM
Unknown error happend executing CeleryScript.
```
Afterwards I tried several reloads of the webpage, but the bot is not logged into the webfrontend nor do I see a network poping up. The last logs were:
```
FARM DESIGNER
CONTROLS
DEVICE
SEQUENCES
REGIMENS
TOOLS
FARMWARE
FB IS LOGGED IN
4:36 PM
FB Setting up rollbar!
4:36 pm
FB needs to restart MQTT
4:36 pm
FB Starting mqtt!
4:36 pm
FB is up and running!
4:36 pm
FB ntp: :ok
4:36 pm
FB is installing first party Farmwares.
4:36 pm
FB is trying to log in.
4:36 pm
FB is logging in…
4:36 pm
FB already has a token. Fetching another.
4:36 pm
FB doing connection test…
4:36 pm
FB connection test complete
4:36 pm
FB starting ntp client.
4:36 pm
FB is getting time from NTP.
4:36 pm
FB trying to set time (try 0)
4:36 pm
FB is waiting for linux and network and what not.
4:36 pm
Sequence complete.
4:35 pm
Doing movement.
```

I needed to unplug the power in order to reset.

**Comment by ConnorRigby** [comment]
I just found this bug. Working on a fix right now.

**Comment by ConnorRigby** [comment]
fixed in 4.0.1
